release-notes-1.4.0.md

```diff
@@ -4,5 +4,6 @@
 ${version-number}
```

Comment:
- Release notes file needs to be renamed to align with the new version number.
- As this is a major version number there should be a "Breaking Changes" section.
Read the Release notes section in https://rdapps.swinfra.net/quixy/#/viewEntity/OCTFT52I355035.